README.md

```diff
@@ -688,6 +688,22 @@ client.without_caching do
 end
 ```
 
+If you prefer to opt in to caching on a per-request, you can do so by using .with_caching and
+setting the use_cache config option to false:
+
+```ruby
+Restforce.configure do |config|
+  config.cache = Rails.cache
+  config.use_cache = false
+end
+```
+
+```ruby
+client.with_caching do
+  client.query('select Id from Account')
+end
+```
+
 Caching is done on based on your authentication credentials, so cached responses will not be shared between different Salesforce logins.
```

Comment:
Looks like an extra 'on' in this sentence - may as well fix while you're at it? 😄